src/pages/Jungmin/Login/InputContainer/SignupInputContainer.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import InputBox from './InputBox';
+import './InputContainer.scss';
+
+class Button extends React.Component {
+  render() {
+    return (
+      <button
+        className={
+          !this.props.disabled
+            ? 'loginButton loginButtonOn'
+            : 'loginButton loginButtonOff '
+        }
```

Comment:
```suggestion
        className={`loginButton ${!disabled ? 'loginButtonOn' : 'loginButtonOff'}`}
```
변하는 부분만 삼항연산자로 표현해줄 수 있습니다